src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -141,10 +141,18 @@ const MainLayout = async (props: Props) => {
           </li>
           <li>
             <a
-              href="https://www.mozilla.org/privacy/firefox-monitor"
+              href="https://www.mozilla.org/en-US/about/legal/terms/subscription-services/"
```

Comment:
nit: remove language specific path and let Bedrock handle best language.
```suggestion
              href="https://www.mozilla.org/about/legal/terms/subscription-services/"
```